utils/preprocess.py

```diff
@@ -49,9 +49,9 @@ def testBinary():
 
 def testDisplayImage():
```

Comment:
add the path as a parameter and do some validation.
`def testDisplayImage(data_dir:`